doc/monitoring/metrics_reference.rst

```diff
@@ -291,3 +291,71 @@ Those metrics provides CPU usage statistics. Only for Linux.
   ``{kind="user", thread_name="tarantool", thread_pid="pid", file_name="init.lua"}``,
   where ``kind`` is ``user`` or ``system``, ``thread_name`` is ``tarantool``, ``wal``,
   ``iproto`` or ``coio``, ``file_name`` is entrypoint file name, e.g. ``init.lua``.
+
+.. _metrics-vinyl:
+
+-------------------------------------------------------------------------------
+Vinyl
+-------------------------------------------------------------------------------
+
+Vinyl metrics provide :ref:`vinyl engine <storing-data-with-vinyl>` statistics.
+
+**Disk**
+
+* ``tnt_vinyl_disk_data_size`` - the amount of data that has gone into files, in bytes
+
+* ``tnt_vinyl_disk_index_size`` - the amount of index that has gone into files, in bytes
+
+* ``tnt_vinyl_disk_data_compacted_size`` - sum size of data stored at the last LSM tree level,
+in bytes, without taking disk compression into account
+
+**Regulator**
+
+* ``tnt_vinyl_regulator_dump_bandwidth`` - the estimated average rate at which dumps are done.
+Initially this will appear as 10485760 (10 megabytes per second). Only significant dumps
+(larger than one megabyte) are used for estimating
+
+* ``tnt_vinyl_regulator_write_rate`` - the actual average rate at which recent writes to disk are done.
+Averaging is done over a 5-second time window
+
+* ``tnt_vinyl_regulator_rate_limit`` - the write rate limit, in bytes per second,
+imposed on transactions by the regulator based on the observed dump/compaction performance
+
+* ``tnt_vinyl_regulator_dump_watermark`` - the point when dumping must occur.
+The value is slightly smaller than the amount of memory that is allocated for vinyl trees,
+which is the vinyl_memory paramete
+
+**Transactional activity**
+
+* ``tnt_vinyl_tx_conflict`` - counts conflicts that caused a transaction to roll back
+
+* ``tnt_vinyl_tx_commit`` - the count of commits (successful transaction ends).
+It includes implicit commits, for example any insert causes a commit unless it is within a begin-end block
+
+* ``tnt_vinyl_tx_rollback`` - the count of rollbacks (unsuccessful transaction ends).
+This is not merely a count of explicit ``box.rollback()`` requests – it includes requests that ended in errors
+
+**Memory**
+
+* ``tnt_vinyl_memory_tuple_cache`` - the number of bytes that are being used for tuples
```

Comment:
The description sounds like "it is about **all** the tuples", while "cache" in metric name implies it is not. Or am I wrong?